### PR TITLE
Initial configuration #1

### LINKED ISSUE #1: Initial working enviroment configuration
# REQUIREMENTS
- Installation of rails 6.1
- Installation gem bootstrap
-  Installation gem device
- Installation gem activeadmin
- Installation gem rspec-rails
- Installation gem factory-bot
- Installation gem database-cleaner
- Install gem simplecov
- install gem annotate
- Installation gem faker
- Postgres configuration

# DESIGN

# Acceptance Criteria
List all required acceptance criteria
- [ ] Installation of rails 6.1
- [ ] Installation gem

### PR DESCRIPTION
**Pull request by Brayan9105**
- Setup the project with Rails 6.1 and postgresql as the default database, this help us to deploy the app in heroku.

-Base configuration has been added with gems like rspec-rails for the testing and other complementing gems for test like factory-bot-rails, database-cleanner and faker, and gems like simplecov to manage the coverage of the test, other gems like annotate help us to show the schema in the model and devise and activeadmin will use to implement a admin module and authentication
 
To colaborate clone the repository and run the next command in the console
rails db:setup
and
bundle install
 
 # Testing
type in console:
rspec

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+group :development do
+  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
+  gem 'web-console', '>= 4.1.0'
+  # Display performance information such as SQL time and flame graphs for each request in your browser.
+  # Can be configured to work on production as well see: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/README.md
+  gem 'rack-mini-profiler', '~> 2.0'
+  gem 'listen', '~> 3.3'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  gem 'annotate', '~> 3.1', '>= 3.1.1'
```
Comment (anxhe) [inline Gemfile:47]: Interesting, I´ll take a note about this gem

File: db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```
Comment (anxhe) [inline db/seeds.rb:8]: We have thought something like this but not only for development

File: test/test_helper.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+ENV['RAILS_ENV'] ||= 'test'
```
Comment (anxhe) [inline test/test_helper.rb:1]: We also forgot this,  we were adviced to delete the test directory as we use rspec

**Review by anxhe — APPROVED** [review]
I think it looks good, only delete test directory before merge.

Good job 🍻

**Review by FranckZnt19 — CHANGES_REQUESTED** [review]
if they are going to start putting functional code, it is necessary to carry out tests of the code

**Review by JorgeDDW — APPROVED** [review]
What is the .keep file for?

**Pull request merged**